test/event_test.js

```diff
@@ -205,9 +205,7 @@ suite('ICAL.Event', function() {
     });
 
     test('with exceptions', function() {
-      subject = new ICAL.Event(primaryItem, {
-        exceptions: exceptions
-      });
+      subject = new ICAL.Event(primaryItem);
```

Comment:
I think it would be good to create a further test, making sure it works with both the automatic method and the manual method. Especially with the previously proposed changes.